src/modules/AutoinstSoftware.rb

```diff
@@ -851,8 +848,12 @@ def Write
       Pkg.PkgApplReset
 
       sw_settings = Profile.current.fetch("software",{})
+      product_name = sw_settings.fetch("product", "")
```

Comment:
I've assumed this is a new element in the profile so I checked the profile schema. Apparently, software / product**s** exists as a list (since 5 years ago).

Reply:
that was my fault. I've used software/product in early development and then forgot to change it. Fixed - using software/products/product